scripts/build-server

```diff
@@ -7,4 +7,13 @@ cd $(dirname $0)/..
 
 mkdir -p bin
 [ "$(uname)" != "Darwin" ] && LINKFLAGS="-linkmode external -extldflags -static -s"
-CGO_ENABLED=0 go build -i -tags k8s -ldflags "-X main.VERSION=$VERSION $LINKFLAGS" -o bin/rancher
+
+while IFS=' ' read -r field1 field2
```

Comment:
can we consider using grep/awk to fetch the needed information from the file?

Reply:
```bash
RKE_VERSION=$(grep '^github.com/rancher/rke' vendor.conf | awk '{print $2}')
```

Please use RKE_VERSION for variable as it matches shell scripting conventions for variables